### ISSUE TITLE
repl command does not support the --unstable option

### ISSUE THREAD

**Issue by srackham** [denoland/deno#5063]
```
$ deno --version
deno 0.42.0
v8 8.2.308
typescript 3.8.3

$ deno repl --unstable
error: Found argument '--unstable' which wasn't expected, or isn't valid in this context
```

The REPL throws you out if you try to execute any unstable APIs:

```
$ deno repl
> Deno.utimeSync('test.txt', new Date(1588561576064), new Date(1588561576064));
Unstable API 'Deno.utime'. The --unstable flag must be provided.

**Comment by zhmushan** [comment]
@srackham see https://github.com/denoland/deno/pull/4957

**Comment by ry** [comment]
Should be fixed in #5026

**Comment by jsejcksn** [comment]
Fix confirmed

**Comment by bartlomieju** [comment]
Fixed in #5026
